pam/integration-tests/vhs-helpers_test.go

```diff
@@ -200,13 +200,11 @@ func newTapeData(tapeName string, outputDir string, settings ...tapeSetting) *ta
 		m[s.Key] = s.Value
 	}
 	return &tapeData{
-		Name:      tapeName,
-		OutputDir: outputDir,
-		Outputs: []string{
-			tapeName + ".txt",
-		},
```

Comment:
The point here was to potentially support producing `gif` or other outputs just adding a line, not sure if we want to lose that if in future we want to generate videos in specific cases.

Reply:
Should be simple enough to add again when we actually need it IMO

Reply:
As you prefer, not a big deal